core/src/test/java/org/apache/iceberg/TestManifestWriterVersions.java

```diff
@@ -22,6 +22,8 @@
 
 import java.io.IOException;
 import java.util.List;
+import java.util.Map;
+import org.apache.iceberg.avro.AvroIterable;
```

Comment:
What is the purpose of the updates here? I can understand testing that the manifest is written with a certain codec, but that should be in `TestManifestWriter` right? This tests different versions, which doesn't have much to do with the compression codec.

Reply:
Sure, I'll move the `ManifestWriter` tests to `TestManifestWriter`.

For `ManifestListWriter` there does not exist a `TestManifestListWriter` class, I'll create the new test class and move the `ManifestListerWriter` tests to it.

Reply:
Done.